configs/pontoon.toml

```diff
@@ -284,3 +284,6 @@ locales = [
 [[paths]]
     reference = "en/products/vpn/shared.ftl"
     l10n = "{locale}/products/vpn/shared.ftl"
+[[paths]]
```

Comment:
The path is fine, but I noticed that there a few out of order. Should we check the entries in this page and move the paths around so they are in the correct order?